components/mmu_server.py

```diff
@@ -77,11 +78,31 @@ def __init__(self, config: ConfigHelper):
             self.server.register_remote_method("spoolman_get_spool_info", self.display_spool_info)
             self.server.register_remote_method("spoolman_display_spool_location", self.display_spool_location)
 
-        # Replace file_manager/metadata with this file
-        self.setup_placeholder_processor(config)
-
         # Options
         self.update_location = self.config.getboolean("update_spoolman_location", True)
+        self.proc_config: Dict[str, Any] = {
+            "name": "mmu_server",
+            "command": " ".join([
+                sys.executable,
+                os.path.abspath(__file__),
+                "-f",
+                "{gcode_file_path}",
```

Comment:
This template value is baked into the new metadata.py: https://github.com/Arksine/moonraker/blob/master/moonraker/components/file_manager/metadata.py#L1065

Its the only value as far as I can tell.